src/Sulu/Bundle/AudienceTargetingBundle/Resources/config/doctrine/TargetGroup.orm.xml

```diff
@@ -35,5 +35,9 @@
                 <cascade-persist/>
             </cascade>
         </one-to-many>
+
+        <indexes>
+            <index name="active_idx" columns="active"/>
+        </indexes>
```

Comment:
Thx, can you move it to the top like here: https://github.com/sulu/sulu/blob/8ed3425211e8b72b4ac7ecc5239842408927fc42/src/Sulu/Bundle/ContactBundle/Resources/config/doctrine/Contact.orm.xml#L8-L11

And we would require a line in the `UPGRADE.md` about the SQL changes.